nhs_number/generate.py

```diff
@@ -76,3 +80,24 @@ def generate(
                 nhs_numbers.append(candidate_str + wrong_checksum_str)
 
     return nhs_numbers
+
+
+def random_chi_str() -> str:
+    """
+    Create nine digits of a CHI number that conforms to the
+    standard format.
+
+    Returns:
+        str: a chi number without checksum digit
+    """
+    start_datetime = datetime.datetime.strptime("1900-01-01", "%Y-%m-%d")
+    end_datetime = datetime.datetime.strptime("1999-12-31", "%Y-%m-%d")
```

Comment:
Why end in 1999? I don't think it an issue but why not 2050 or somesuch?

Reply:
Because we are only interested in the last two digits of the year and 1900-1999 covers all possible numbers.

Reply:
If you have someone born in 1920 and someone born in 2020 does everything just work the same?